examples/rpc-custom-api/src/example.ts

```diff
@@ -76,8 +78,8 @@ const customizedRpcApi = new Proxy(solanaRpcApi, {
             /**
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +75]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->```ts
    get(target, p, receiver): (address: Address) => RpcPlan<AssetMetadata> {
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/3442?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->